pkg/orchestration/wiring/testdata/ec2compute.overwrite.state.yaml

```diff
@@ -20,9 +20,6 @@ spec:
         maxSize: 2
       notifications:
         email: abc@email.com
-        pagerduty:
```

Comment:
we are not going to support env overrides for alarms since it would be solved in OpsGenie alarm routing rules